tests/unit/s2n_tls13_handshake_test.c

```diff
@@ -144,9 +144,14 @@ int main(int argc, char **argv)
         EXPECT_EQUAL(s2n_stuffer_data_available(&client_conn->out), 9);
         EXPECT_SUCCESS(s2n_stuffer_wipe(&client_conn->out));
         client_conn->client = &client_conn->secure;
-
+        uint8_t expected_encrypted_bytes_out;
+        uint8_t mac_digest_size;
+        EXPECT_SUCCESS(s2n_hmac_digest_size(client_conn->client->client_record_mac.alg, &mac_digest_size));
+        expected_encrypted_bytes_out = client_conn->encrypted_bytes_out + mac_digest_size +  TLS13_CONTENT_TYPE_LENGTH + 
```

Comment:
Rather than incorporate expected_encrypted_bytes_out into this functional test, can we write a specific unit test? Or is the setup for sending a record via s2n_record_write too involved?